### ISSUE TITLE
"This QueryDict instance is immutable"

### ISSUE THREAD

**Issue by willseward** [barseghyanartur/django-fobi#66]
Django==1.11.0
fobi==0.11.8

```text
Traceback:  

File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/exception.py" in inner
 41.             response = get_response(request)

File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py" in _get_response
 187.                 response = self.process_exception_by_middleware(e, request)

File "/usr/local/lib/python2.7/dist-packages/django/core/handlers/base.py" in _get_response
 185.                 response = wrapped_callback(request, *callback_args, **callback_kwargs)

File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py" in view
 68.             return self.dispatch(request, *args, **kwargs)

File "/usr/local/lib/python2.7/dist-packages/fobi/wizard/views/dynamic.py" in dispatch
 332.             request, *args, **kwargs

File "/usr/local/lib/python2.7/dist-packages/django/views/generic/base.py" in dispatch
 88.         return handler(request, *args, **kwargs)

File "/usr/local/lib/python2.7/dist-packages/fobi/views.py" in post
 1581.                     form.data[wizard_form_key] = field_value

File "/usr/local/lib/python2.7/dist-packages/django/http/request.py" in __setitem__
 434.         self._assert_mutable()

File "/usr/local/lib/python2.7/dist-packages/django/http/request.py" in _assert_mutable
 431.             raise AttributeError("This QueryDict instance is immutable")

Exception Type: AttributeError at /wizard-view/merchant-onboard/
Exception Value: This QueryDict instance is immutable
Request information:
USER: AnonymousUser
```

This happens when POSTing a form sometimes. I can't find a reproducible setup.

**Comment by barseghyanartur** [comment]
@willseward:

Thanks for reporting this. Thus, for your experience, does this happen in form-wizards only? Or have you experienced it on normal forms (or DRF integration) as well?

**Comment by willseward** [comment]
Just on wizard forms. 

I was able to resolve the issue with this: https://github.com/GivePay/django-fobi/commit/1c797e35767b6d1ee3c52930b0598a35b757a520#diff-27fd462ed7c61a3988f0a5260547dd67R1581

Of course, that logic should probably be moved to above the constructor for the form.

```python
self.request.POST._mutable = True
```

**Comment by barseghyanartur** [comment]
@willseward:

Thanks! I'll fix it tonight.

**Comment by barseghyanartur** [comment]
@willseward:

Thanks again! Fixed in version [0.11.10](https://pypi.python.org/pypi/django-fobi/0.11.10).
